lib/Core/BuildEngine.cpp

```diff
@@ -965,6 +965,8 @@ class BuildEngineImpl {
     
     // Find the cycle by searching from the entry node.
     struct WorkItem {
+      WorkItem(Rule * node) { this->node = node; }
+
```

Comment:
I don't know, I would need to read the standard for the uniform initializer behavior to understand why there might be a divergence here.

I suspect that `WorkItem{ 0, 0 }` works on both compilers?